README.md

```diff
@@ -20,9 +20,13 @@ version that is being released.
 
 To cut a new release:
 
-1. Create a new tag, e.g. `git tag x.y.z`
-2. Push the tag up `git push origin x.y.z`
-3. Kick off the release job hosted in the shared tools account (e.g. `publish-aws-kotlin-repo-tools`) specifying the tag as the source version to build.
+1. Create a new tag, e.g. `git tag x.y.z`.
+2. Push the tag up `git push origin x.y.z`.
+3. Go to the CodeBuild release job hosted in the shared tools account (e.g. `publish-aws-kotlin-repo-tools`).
+4. Start a build with overrides.
+5. Under source connect with your GitHub account.
```

Comment:
This could be more clear: "Under Source -> Connection status, make sure your GitHub account is connected."